src/docs/truffle/getting-started/running-migrations.md

```diff
@@ -24,15 +24,21 @@ A simple migration file looks like this:
 Filename: `4_example_migration.js`
 
 ```javascript
-var MyContract = artifacts.require("MyContract");
+const MyContract = artifacts.require("MyContract");
 
-module.exports = function(deployer) {
+module.exports = async function (deployer, network, accounts) {
   // deployment steps
-  deployer.deploy(MyContract);
+  await deployer.deploy(MyContract);
 };
 ```
 
-Note that the filename is prefixed with a number and is suffixed by a description. The numbered prefix is required in order to record whether the migration ran successfully. The suffix is purely for human readability and comprehension.
+Note that the filename is prefixed with a number and is suffixed by a description. The numbered prefix is required in order to record whether the migration ran successfully. The suffix is purely for human readability and comprehension. Your migration function also may
+be `async` if you wish (as written above) in order to use the `await` keyword to await your deployments.
+
+Another thing to note is that each migration function takes 3 arguments:
+1. `deployer` which is the object responsible for deploying contracts
+2. `network` which is the name (string) of the network being used during the migration
+3. `accounts` which is an array of the available (unlocked accounts) during the migration
```

Comment:
Is the parenthesis necessary?

Reply:
I made a mistake here - I meant to only have "unlocked" in parens.